### ISSUE TITLE
Sentry responded with an error: HTTP 413: Request Entity Too Large

### ISSUE THREAD

**Issue by dylanbfox** [getsentry/raven-python#981]
I have an exception being raised in a POST request handler in Tornado. When that exception is sent to Sentry via `self.captureException(True)`, I am seeing:

```
ERROR:sentry.errors:Sentry responded with an error: HTTP 413: Request Entity Too Large (url: https://sentry.io/api/148507/store/)
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/raven/contrib/tornado/__init__.py", line 70, in _handle_result
    future.result()
  File "/usr/local/lib/python2.7/dist-packages/tornado/concurrent.py", line 237, in result
    raise_exc_info(self._exc_info)
  File "<string>", line 3, in raise_exc_info
HTTPError: HTTP 413: Request Entity Too Large
```

In the POST body is a large file. I tried to exclude the POST body data by adding the `'raven.processors.RemovePostDataProcessor'`, but I am still getting the 413 response.

```
app.sentry_client = AsyncSentryClient(_sentry_dsn, 
	processors=(
		'raven.processors.SanitizePasswordsProcessor',
		'raven.processors.RemovePostDataProcessor'
	)
)
```

I am using `raven==6.0.0`.

**Comment by dylanbfox** [comment]
This has something to do with the automatic breadcrumbs, not the POST body. I turned off the default breadcrumbs and the issue went away. The function that raised the exception which was throwing the 413 error was calling multiple other functions and passing a lot of data around. Maybe that generated large breadcrumbs somehow?

**Comment by pramod-hegde** [comment]
Where are you hosting the service? This certainly needs few configuration changes on the server.

**Comment by sergei-maertens** [comment]
I've also just run into this, looks like breadcrumbs are causing 413s. Sentry itself is self-hosted, and we're using Django integration.

However, it looks like there's no setting to disable breadcrumbs in a Django context, or am I looking at the wrong place?

**Comment by peterbe** [comment]
Can this issue be closed in favor of https://github.com/getsentry/raven-python/issues/604
which is old but has some fresh activity from @mitsuhiko

**Comment by ashwoods** [comment]
Closed as duplicate #604
